packages/frontend/components/Form/ChainSelect.tsx

```diff
@@ -68,22 +77,35 @@ export default function ChainSelect() {
         MenuListProps={{ "aria-labelledby": "basic-button" }}
         anchorOrigin={{ vertical: "bottom", horizontal: "left" }}
         TransitionComponent={Fade}
-        sx={{ marginTop: 1 }}
+        sx={{
+          marginTop: 1,
+          ".MuiPaper-root": {
```

Comment:
I themed the menu in main branch, so you don't need to theme it again 😋 

https://github.com/Fujicracy/fuji-v2/blob/56090503d1b6f12edbcf3e83c703d03fe48c0442/packages/frontend/styles/theme.ts#L377